application/src/main/java/org/togetherjava/tjbot/commands/basic/UpTimeCommand.java

```diff
@@ -0,0 +1,69 @@
+package org.togetherjava.tjbot.commands.basic;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.internal.utils.Checks;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+
+import java.lang.management.ManagementFactory;
+
+/**
+ * Informs the use how long the bot has been running for.
+ */
+public class UpTimeCommand extends SlashCommandAdapter {
+    /**
+     * Constructs an instance.
+     */
+    public UpTimeCommand() {
+        super("uptime", "Tells the user how long the bot has been up for",
```

Comment:
the description is bad UX, this is talking to the user and not to the programmer. rewrite